test/unit/data_hygiene/policy_tagger_test.rb

```diff
@@ -0,0 +1,81 @@
+require "test_helper"
+require "data_hygiene/policy_tagger"
+
+class PolicyTaggerTest < ActiveSupport::TestCase
+  setup do
+    @content_id_1 = SecureRandom.uuid
+    @content_id_2 = SecureRandom.uuid
+    @published_edition = create(:published_publication)
+    @document = @published_edition.document
+    @draft_edition = create(:draft_publication, document: @document)
+    stub_registration
+  end
+
+  test "updates policies for all editions of the document" do
+    # adding
+    policies_to_add = [@content_id_1, @content_id_2]
+    PolicyTagger.new(
+      slug:               @document.slug,
+      policies_to_remove: [],
```

Comment:
Maybe this keyword argument on the initializer could default to `[]` so you could omit these in this test and subsequent ones?